spring-cloud-gcp-autoconfigure/src/test/java/com/google/cloud/spring/autoconfigure/trace/pubsub/TracePubSubAutoConfigurationTest.java

```diff
@@ -86,13 +86,13 @@ void test() {
 
   @Test
   void testPubSubTracingDisabledByDefault() {
-    this.contextRunner.run(
-        context -> {
-          assertThat(context.getBeansOfType(TracePubSubBeanPostProcessor.class)).isEmpty();
-          assertThat(context.getBeansOfType(PubSubTracing.class)).isEmpty();
-          assertThat(context.getBeansOfType(MessagingTracing.class)).isEmpty();
-          assertThat(context.getBeansOfType(ObservedAspect.class)).isEmpty();
-        });
```

Comment:
[Note for future reference] As discussed offline, this class is provided by Spring by default.